.release-notes/4547.md

```diff
@@ -0,0 +1,7 @@
+##  Add support for pinning actors to a dedicated scheduler thread
+
+Pony programmers can now pin actors to a dedicated  scheduler thread. This can be required/used for interfacing with C libraries that rely on thread local storage. A common example of this is graphics/windowing libraries.
+
+The way it works is that an actor can request that it be pinned (which may or may not happen immediately) and then it must wait and check to confirm that the pinning was successfully applied (prior to running any workload that required the actor to be pinned) after which all subsequent behaviors on that actor will run on the same scheduler thread until the actor is destroyed or the actor requests to be unpinned.
+
+Additional details can be found in the `actor_pinning` package in the standard library.
```

Comment:
I like "robust" release notes. Can you add an example program in here that demonstrates the concepts?

I also think the release notes should include a notice that this is an experimental feature and likely to change. That the implementation will evolve and that might change the feature in general. That sort of thing.

Reply:
the release notes have been fattened up..